src/skins/o2-new.tsx

```diff
@@ -36,7 +36,7 @@ export const palette = {
     o2Red40: '#FF8B8B',
```

Comment:
Some elements have little contrast (the "3" in the stepper or the "check" in the checkbox):
![image](https://github.com/Telefonica/mistica-web/assets/579417/0e3b45ac-c60b-4a3c-ba97-fe2bc170bec9)
![image](https://github.com/Telefonica/mistica-web/assets/579417/02debbe4-216b-4f71-87e0-6c05b019d2c3)

Reply:
Yea, we know. Probably we need to change these elements to `background` token to ensure that. But with this PR I would like to avoid design changes